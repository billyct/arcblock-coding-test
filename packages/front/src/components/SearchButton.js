import cn from 'classnames'
import {ReactComponent as SearchIcon} from '../assets/search.svg'

const SearchButton = ({className, ...rest}) => (
  <button
    className={cn(
      'px-4 hover:bg-black hover:text-white outline-none focus:outline-none',
      className,
    )}
    {...rest}
  >
    <SearchIcon width={20} height={20}/>
  </button>
)
export default SearchButton
