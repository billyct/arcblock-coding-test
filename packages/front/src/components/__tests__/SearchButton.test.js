import {render} from '@testing-library/react'

import SearchButton from '../SearchButton'

test('SearchButton Component', () => {
  const {container} = render(<SearchButton/>)
  expect(container).toMatchSnapshot()
})
