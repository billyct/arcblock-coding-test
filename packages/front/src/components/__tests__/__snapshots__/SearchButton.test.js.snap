// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchButton Component 1`] = `
<div>
  <button
    class="px-4 hover:bg-black hover:text-white outline-none focus:outline-none"
  >
    <svg
      height="20"
      width="20"
    >
      search.svg
    </svg>
  </button>
</div>
`;
